lib/posix-time/Config.uk

```diff
@@ -0,0 +1,4 @@
+config LIBPOSIX_TIME
+       bool "posix-time: Time syscalls"
+       default n
+       select HAVE_TIME
```

Comment:
This should change now to `HAVE_POSIX_TIME`, right? + all the subsequent changes

This is something pretty minor so I wouldn't block on it

Reply:
It is another symbol that is defined. Probably there are sources that depend on the definition of this name. I would not rename it because errors may occur in the replacement process.

Reply:
Ok, will mark as resolved